lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1833,6 +1831,34 @@ sub test_resultfile_list {
     return \@filelist_existing;
 }
 
+=head2 investigate
+
+Find pointers for investigation on failures, e.g. what changed vs. a "last
+good" job in the same scenario.
+
+=cut
+sub investigate {
+    my ($self, %args) = @_;
+    my %investigation;
+    my $user_id  = $self->result_source->schema->resultset('Users')->search({username => 'system'})->first->id;
+    my %msg      = (text => "## Investigation hints\n", user_id => $user_id);
+    my @previous = $self->_previous_scenario_jobs;
+    return $self->comments->create(
+        {%msg, text => $msg{text} . 'No previous job in this scenario, cannot provide hints'})
```

Comment:
Please don't abuse comments for this. Provide this on request

Reply:
of course we can. But we also use comments for labels so I thought this would be fitting.

When we provide it on request, ideas how to integrate? I thought of a tab which loads this on demand

Reply:
I would integrate a popup into 'next & previous' with loading the details on demand.

Reply:
We decided for a tab with loading-on-demand for now which is only shown for "failed" jobs, mainly for the following reasons:

* It's easier to find for users
* Copy-pasting something from content in a tab is easier than a popup
* I would like to extend information so I guess it would become too much content